ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -109,6 +109,7 @@ Wait for RHODS Dashboard to Load
     IF    ${wait_for_cards} == ${TRUE}
         Wait Until Cards Are Loaded
     END
+    [Teardown]   Run Keyword If Test Failed    Capture Page Screenshot
```

Comment:
why are we adding this in the Teardowns? Shouldn't be something that has to be done by default if the test fails?

Reply:
Yes, you're right. But it doesn't capture the screenshot in test setup...
I think it's a bug in RF Selenium Library, so I opened an issue for them: https://github.com/robotframework/SeleniumLibrary/issues/1887

Reply:
I've reverted this change, as it was caused by protected kw that hides screenshots.